etc/ci/upload_nightly.sh

```diff
@@ -12,7 +12,12 @@ usage() {
 
 
 upload() {
-    s3cmd put "${2}" "s3://servo-builds/nightly/${1}/"
+    local nightly_upload_dir package_filename package_upload_path
```

Comment:
We declare `nightly_upload_dir` on the next line, so we don't need it on this line.